route/doc.go

```diff
@@ -184,10 +184,10 @@ point into the higher dimension by appending 0s.
 
 Indexed Measures
 
-Models that do not require points operate on indices. These indices may or
```

Comment:
I believe indices is correct. Maybe a native speaker has a different opinion? [See here](https://english.stackexchange.com/questions/61080/the-plural-of-index-indexes-or-indices)

Reply:
Let's go for indices then